server/server.js

```diff
@@ -172,6 +184,40 @@ app.post('/genotet/upload', upload.single('file'), function(req, res) {
   });
 });
 
+/**
+ * POST request is not used as it conflicts with jsonp.
```

Comment:
Anything reasonably explains what app.post is doing here (handling post requests from client such as blabla). Besides we are no longer using jsonp.